sr_vision_mocks/src/sr_vision_mocks/mock_known_objects.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2017 Shadow Robot Company Ltd - All Rights Reserved.
+# Proprietary and Confidential. Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+
+import rospy
+import tf
+import re
+from sr_manipulation_msgs.msg import AvailableObjects, GraspObject
+from geometry_msgs.msg import Pose
+from threading import Lock
+
+
+class MockKnownObjectsPublisher(object):
+    def __init__(self):
+        self.object_regex = rospy.get_param("~object_regex", ".*duplo_2x4x1_")
```

Comment:
do we really want to default to duplo values? it is nice however to have an example. maybe a comment before each line of what is expected of each parameter